configuration/configuration_reference.md

```diff
@@ -10,11 +10,20 @@
                 <a href="#bases">&lt;bases/&gt;</a>
                     <a href="#base">&lt;base/&gt;</a>
             <a href="#passwordFile">&lt;passwordFile/&gt;</a>
+            <a href="#authConfigs">&lt;authConfigs&gt;</a>
```

Comment:
As far as xml structure is concerned, this looks wrong to me as `ldap` tag contains `bases` which contains `base` so it cannot close in the same line:
```
<ldap/>
       <bases/>
             <base/>
``` 
Good that you noticed this. The one for authConfig looks proper, so I am fixing the ldap and password file entries instead.